scheduler/admin.py

```diff
@@ -110,8 +110,7 @@ def run_job_now(self, request, queryset):
         job_names = []
         for obj in queryset:
             kwargs = obj.schedule_kwargs()
-            obj.get_queue2().enqueue_at(
-                utc(now()),
+            obj.get_queue2().enqueue(
```

Comment:
Please revert this change and open an issue. I might address the "force run now" differently.

Reply:
Thanks! Will do.
I wonder why there's a need to enqueue_at now, if enqueue does that without further requiring the scheduler(=worker) to put it in the queue.

Reply:
@cunla I will make a new PR because I need the current branch because I'm already referencing it.